src/sequelize/models/User.js

```diff
@@ -62,8 +62,8 @@ export default (sequelize, DataTypes) => {
     }
   );
 
-  User.associate = () => {
-    // Association definitions here
+  User.associate = (models) => {
+    User.hasMany(models.Article, { foreignKey: 'authorId', as: 'author' });
```

Comment:
add the `onDelete: 'CASCADE'` setting so that articles belonging to a user are also deleted when the user is deleted

Reply:
done.